src/main.cpp

```diff
@@ -599,5 +633,10 @@ int main(int argc, char **argv)
 	}
 	cout << "Device is now registered." << endl;
 
+	if (options.start_daemon) {
+		cout << "Starting aktualizr-lite daemon" << endl;
+		_spawn("systemctl start aktualizr-lite");
```

Comment:
@ricardosalveti What do we think about using `systemctl restart aktualizr-lite` here.  That would cover an edge case where a user uses this while it's running.

Reply:
I think @doanac added an extra logic that aborts in case it is running, so start would work fine.